pkg/reconciler/common/workload_override.go

```diff
@@ -61,6 +62,14 @@ func OverridesTransform(overrides []base.WorkloadOverride, log *zap.SugaredLogge
 					ss.Spec.Replicas = override.Replicas
 				}
 			}
+			if u.GetKind() == "Job" && u.GetGenerateName() == override.Name {
+				ss := &batchv1.Job{}
```

Comment:
```suggestion
				job := &batchv1.Job{}
```
nit: `ss` is used for `StatefulSet` above but here is job.